main/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,77 @@
+import AccordionItem, {AccordionData} from './AccordionItem';
+import {Animated, StyleProp, ViewStyle} from 'react-native';
+import React, {FC, ReactElement, useRef} from 'react';
+
+import styled from '@emotion/native';
+
+const Container = styled.View`
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const TOGGLE_ELEMENT_PROPERTY = 'toggleElement';
+export interface Styles {
+  titleContainer?: StyleProp<ViewStyle>;
+  bodyContainer?: StyleProp<ViewStyle>;
+}
+
+export interface AccordionBaseProps<T> {
+  data: T;
+  style?: StyleProp<ViewStyle>;
+  styles?: Styles;
+  shouldAnimate?: boolean;
+  collapseOnStart?: boolean;
+  animDuration?: number;
+  activeOpacity?: number;
+  [TOGGLE_ELEMENT_PROPERTY]?: ReactElement | null;
```

Comment:
I think this is redundant. Why not just `toggleElement` here?

Reply:
Currently it's provided to ```AccordionBaseProps``` and parameter of ```renderIndicator``` function at AccordionItem.
I think it have a possibility of change so i separated it into variables. [ref](https://github.com/dooboolab/dooboo-ui/issues/107#issuecomment-917349119)
and this type definition is used only inside the component.
if it's weird please give me advice! thanks.

Reply:
@luke-hanwook I am still not getting this. Why can't we change this to https://github.com/dooboolab/dooboo-ui/commit/f4e2a6e8de3153da5ad6b18d17bcff62a7fc2232?

Reply:
@hyochan
Could you change it to optional property?

Reply:
Yes optional is still fine~!